lib/tasks/import.rake

```diff
@@ -28,4 +28,47 @@ namespace :import do
     puts "Missing ETDs: #{missing_etds.inspect}" if missing_etds.present?
     puts "Import Complete #{Time.zone.now}"
   end
+
+  task :adrl_arks, [:file_path] => :environment do |_t, args|
+    puts "Beginning import #{Time.zone.now}"
+    # CSV file contents would be
+    # merritt,proquest
+    # ark:/13030/m00999g9,ProQuestID:0035D
+    csv = CSV.read(args[:file_path], headers: true)
+    count = 0
+    missing_files = non_adrl = []
+
+    # rewrite CSV with each run
+    # using 'w' mode
+    CSV.open("tmp/adrl_arks.csv",
```

Comment:
is this path going to be okay in production, or should we make this configurable? `args.fetch(:out_path, 'tmp/adrl_arks.csv')`?